akka-persistence-typed-tests/src/test/scala/akka/persistence/typed/scaladsl/EventSourcedBehaviorStashSpec.scala

```diff
@@ -325,9 +325,10 @@ class EventSourcedBehaviorStashSpec
 
       unhandledProbe.receiveMessages(10)
 
-      val value1 = stateProbe.expectMessageType[State](5.seconds).value
-      val value2 = stateProbe.expectMessageType[State](5.seconds).value
-      val value3 = stateProbe.expectMessageType[State](5.seconds).value
+      // a lot of events, so give it some extra time to complete
+      val value1 = stateProbe.expectMessageType[State](10.seconds).value
+      val value2 = stateProbe.expectMessageType[State](10.seconds).value
+      val value3 = stateProbe.expectMessageType[State](10.seconds).value
```

Comment:
This test case simply writes a lot of events, giving it more time fixed the issue